content/03-reference/01-tools-and-interfaces/02-prisma-client/08-pagination/index.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: 'Pagination'
+metaTitle: 'Pagination (Reference)'
+metaDescription: 'Prisma Client provides features that are not achievable with relational databases. These features are referred to as "polyfills" and explained on this page.'
+---
+
+## Overview
+
+Prisma Client supports both offset and cursor-based pagination.
+
+## Offset pagination
+
+Offset pagination combines `skip` and `take` to skip a certain number of results and select a limited range. The following query skips the first 3 `Post` records and returns records 4 - 7:
+
+```ts line-number
+const results = await prisma.post.findMany({
+  skip: 3,
+  take: 4,
+})
+```
+
+![](pagination/offset-skip-take.png)
+
```

Comment:
Something like this might be handy here:
```suggestion

To implement pages of results, you would just `skip` the number of pages multiplied by the number of results you show per page.

```